streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -321,15 +342,15 @@ private void verifyAllWindowedKeys(final List<KafkaStreams> streamsList,
 
             for (final String key: keys) {
                 try {
-                    final StreamsMetadata metadata = streams
-                        .metadataForKey(storeName, key, new StringSerializer());
-                    if (metadata == null || metadata.equals(StreamsMetadata.NOT_AVAILABLE)) {
+                    final KeyQueryMetadata metadata = streams
+                        .queryMetadataForKey(storeName, key, new StringSerializer());
```

Comment:
I pondered this early on. Seems some deprecated methods in KafkaStreams, just got their tests changed. I could call the other method as well `metadataForKey` and match it with the new API may be, to prevent just duplicating tests. wdyt

Reply:
```
/Users/vchandar/Code/kafka/streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java:352: warning: [deprecation] <K>metadataForKey(String,K,Serializer<K>) in KafkaStreams has been deprecated
```
ah yes. build fails when deprecated methods are called. Let me see if there is a way to supress this and have one call in the integration test that validates the new and old api against each other